crates/ui/src/select.rs

```diff
@@ -647,6 +647,11 @@ where
             .map(|item| item.value().clone());
     }
 
+    pub fn delegate<'a>(&self, cx: &'a App) -> &'a D {
```

Comment:
This API not match with the `List::delegate` and `delegate_mut` is also not.

So I don't think we can just add this one is good.